src/style/defaultTheme.ts

```diff
@@ -0,0 +1,302 @@
+import {
+  css,
+  Breakpoints,
+  ColorTheme,
+  CommonStyleProps,
+  FontTheme,
+  SerializedStyles,
+  Theme,
+} from '.'
+
+const normalizeMin =
+  '/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */html{line-height:1.15;-webkit-text-size-adjust:100%}body{margin:0}main{display:block}h1{font-size:2em;margin:.67em 0}hr{box-sizing:content-box;height:0;overflow:visible}pre{font-family:monospace,monospace;font-size:1em}a{background-color:transparent}abbr[title]{border-bottom:none;text-decoration:underline;text-decoration:underline dotted}b,strong{font-weight:bolder}code,kbd,samp{font-family:monospace,monospace;font-size:1em}small{font-size:80%}sub,sup{font-size:75%;line-height:0;position:relative;vertical-align:baseline}sub{bottom:-.25em}sup{top:-.5em}img{border-style:none}button,input,optgroup,select,textarea{font-family:inherit;font-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}[type=button],[type=reset],[type=submit],button{-webkit-appearance:button}[type=button]::-moz-focus-inner,[type=reset]::-moz-focus-inner,[type=submit]::-moz-focus-inner,button::-moz-focus-inner{border-style:none;padding:0}[type=button]:-moz-focusring,[type=reset]:-moz-focusring,[type=submit]:-moz-focusring,button:-moz-focusring{outline:1px dotted ButtonText}fieldset{padding:.35em .75em .625em}legend{box-sizing:border-box;color:inherit;display:table;max-width:100%;padding:0;white-space:normal}progress{vertical-align:baseline}textarea{overflow:auto}[type=checkbox],[type=radio]{box-sizing:border-box;padding:0}[type=number]::-webkit-inner-spin-button,[type=number]::-webkit-outer-spin-button{height:auto}[type=search]{-webkit-appearance:textfield;outline-offset:-2px}[type=search]::-webkit-search-decoration{-webkit-appearance:none}::-webkit-file-upload-button{-webkit-appearance:button;font:inherit}details{display:block}summary{display:list-item}template{display:none}[hidden]{display:none}'
+
+const defaultColorTheme: ColorTheme = {
+  accent: {
+    dark: '#077C7D',
+    normal: '#00B7B8',
+    light: '#6AE1E2',
+  },
+  canvas: {
+    dark: '#E9E9E9',
+    normal: '#FAFAFA',
+    light: '#FFF',
+  },
+  danger: {
+    dark: '#C62828',
+    normal: '#F44336',
+    light: '#FFEBEE',
+  },
+  ink: {
+    dark: '#000',
+    normal: '#444',
+    light: '#888',
+  },
+  primary: {
+    dark: '#054B2',
+    normal: '#0078D4',
+    light: '#1499FF',
+  },
+  success: {
+    dark: '#F8800',
+    normal: '#8BC34A',
+    light: '#C5DBB2',
+  },
+  warning: {
+    dark: '#C5C00',
+    normal: '#E65100',
+    light: '#FFE0B2',
+  },
+}
+
+const defaultFontTheme: FontTheme = {
+  sans: 'sans-serif',
+  serif: 'serif',
+  mono: 'monospace',
+  accent: 'sans-serif',
+  text: {
+    css: ({ color, font }: Theme): SerializedStyles => css`
+      /*
+      Setting this in HTML so divs, spans and other elements benefit
+      from these rules
+      */
+      html {
+        color: ${color.ink.normal};
```

Comment:
I'd remove anything that has too much to do with font and leave it to whatever normalize chooses  It's more clear / better documented to define them in the individual websites.